templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                 <a class="btn btn-primary" href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a class="btn btn-danger" href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">Delete</a>
```

Comment:
Similarly, change `manufacturer.id` to `manufacturer.pk` in the URL tag for consistency and to follow Django's convention for primary keys.